swap/src/network/cbor_request_response.rs

```diff
@@ -45,9 +45,9 @@ where
             e => io::Error::new(io::ErrorKind::Other, e),
         })?;
         let mut de = serde_cbor::Deserializer::from_slice(&message);
-        let msg = Req::deserialize(&mut de).map_err(|e| {
-            tracing::debug!("serde read_request error: {:?}", e);
-            io::Error::new(io::ErrorKind::Other, e)
+        let msg = Req::deserialize(&mut de).map_err(|error| {
+            tracing::debug!(%error, "serde read_request error");
```

Comment:
I think these can just go away, we are passing the error on so we are likely just gonna end up logging them twice.

Reply:
Do you mean the whole log message should go away? e.g.
```suggestion
```

Reply:
You can check if we properly log it upstream, but I assume we are. So yes, the complete log statement can be deleted here.